ethereum/contracts/interfaces/relayer/IWormholeRelayer.sol

```diff
@@ -142,6 +399,21 @@ interface IWormholeRelayerSend is IWormholeRelayerBase {
         uint8 consistencyLevel
     ) external payable;
 
+    /**
+     * @notice Requests a previously published delivery instruction to be redelivered 
```

Comment:
I think we want to spell out the primary use case: To switch delivery providers in case of a non-delivery or in case of delivery being too slow. Otherwise the natural question would be "why on god's green earth would I ever want to redeliver anything?"